app/pb_kits/playbook/pb_person/person.rb

```diff
@@ -2,55 +2,17 @@
 
 module Playbook
   module PbPerson
-    class Person < Playbook::PbKit::Base
-      PROPS = %i[
-        configured_classname
-        configured_data
-        configured_id
-        configured_first_name
-        configured_last_name
-      ].freeze
+    class Person
+      include Playbook::Props
 
-      def initialize(
-          classname: default_configuration,
-          data: default_configuration,
-          id: default_configuration,
-          first_name: default_configuration,
-          last_name: default_configuration
-        )
-        self.configured_classname = classname
-        self.configured_data = data
-        self.configured_id = id
-        self.configured_first_name = first_name
-        self.configured_last_name = last_name
-      end
-
-      def first_name
-        pb_first_name = Playbook::PbBody::Body.new(tag: "span", classname: "pb_person_first") do
-          default_value(configured_first_name, "")
-        end
-        ApplicationController.renderer.render(partial: pb_first_name, as: :object)
-      end
-
-      def last_name
-        if is_set? configured_last_name
-          pb_last_name = Playbook::PbTitle::Title.new(text: configured_last_name, size: 4)
-          ApplicationController.renderer.render(partial: pb_last_name, as: :object)
-        end
-      end
-
-      def to_partial_path
-        "pb_person/person"
-      end
+      partial "pb_person/person"
 
-    private
+      prop :first_name
+      prop :last_name
 
-      DEFAULT = Object.new
-      private_constant :DEFAULT
-      def default_configuration
-        DEFAULT
+      def classname
+        generate_classname("pb_person_kit", first_name, last_name)
```

Comment:
This should not be taking `first_name` and `last_name`. It should only be `generate_classname("pb_person_kit")`